src/libsync/excludedfiles.h

```diff
@@ -60,13 +63,10 @@ public slots:
     /**
      * Reloads the exclude patterns from the registered paths.
      */
-    void reloadExcludes();
+    bool reloadExcludes();
 
 private:
-    ExcludedFiles();
-    ~ExcludedFiles();
-
-    c_strlist_t* _excludes;
+    c_strlist_t** _excludesPtr;
```

Comment:
could there be a comment explaining what this pointer is to and what is the ownership and life time of it